### PR TITLE
Use a custom settings.xml that uses our internal nexus as a mirror of…

### PR DESCRIPTION
**Pull request by iocanel**
… central and jcenter.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,14 +75,14 @@
   </distributionManagement>
 
   <repositories>
-    <repository>
+    <!--repository>
```
Comment (jimmidyson) [inline pom.xml]: Doesn't this mean local builds will fail?

Comment (iocanel) [inline pom.xml]: If bintray is used just as a central mirror, everything should be fine.

It seems though, that the swagger plugin depends on stuff that are exclusive to bintray, so we have to use it and provide an additional mirror for that in nexus.

Comment (iocanel) [inline pom.xml]: In other words, we need to uncomment that and make sure our nexus image provides a bintray proxy.

Comment (jimmidyson) [inline pom.xml]: Agreed. So should we close this PR then?

**Comment by iocanel** [comment]
retest this please

**Pull request merged**